### PR TITLE
[image] Add webP support

### PR DESCRIPTION
**Pull request by jgonet**
# Why

To extend the range of supported formats.

# How

Supporting webP boiled down to adding new codecs for Glide and SDWebImage. Both use libwebp for decoding.
The web version should be supported [natively](https://caniuse.com/webp).

I added new test files using git LFS.

In the future, we probably want to make additional codecs opt-in in order to not increase app size when certain formats aren't used.

# Test Plan

Tested on bare-expo.

https://user-images.githubusercontent.com/12465392/123523749-00c96780-d6c6-11eb-856d-c120dbb5a2a3.mov

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.io and README.md).
- [ ] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: packages/expo-image/android/build.gradle [hunk 1]
```diff
@@ -73,15 +73,18 @@ repositories {
     url "$rootDir/../node_modules/jsc-android/dist"
   }
   google()
-  jcenter()
+  mavenCentral()
 }
 
 dependencies {
+  def GLIDE_VERSION = "4.12.0"
```
Comment (bbarthec) [inline packages/expo-image/android/build.gradle:80]: I'm not sure I'm a fan of this approach 🤔 
Give me some arguments for extracting the library version to a variable 🤔 
My argument that counters this approach is: `in package.json you write every library version next to the library name` (yeah, I know json does not allow variables but still...) 🤔

File: .gitattributes [hunk 2]
```diff
@@ -4,3 +4,4 @@ apps/bare-expo/ios/BareExpo.xcodeproj/project.pbxproj merge=union -text diff lin
 packages/*/build/** -diff linguist-generated
 packages/*/plugin/build/** -diff linguist-generated
 packages/@*/*/build/** -diff linguist-generated
+*.webp filter=lfs diff=lfs merge=lfs -text
```
Comment (bbarthec) [inline .gitattributes:7]: Do we need this one? 🤔

**Review by bbarthec — APPROVED** [review]
Looks fine to me 👍 
Great job! 😊 🎉

File: packages/expo-image/android/build.gradle (hunk 1 above)
Comment (jakub-gonet) [reply packages/expo-image/android/build.gradle:80]: I'm a fan of parametrizing things that should be together - here we have a base glide library in some version that has to be in sync with annotation processor and plugin version. That's exactly the use-case for variables, allowing changing contents everywhere by doing it in one place. It's local as well so I don't see any problem here.

File: .gitattributes (hunk 2 above)
[the remark this answers is not in the document]
Comment (jakub-gonet) [reply .gitattributes:7]: Sure, I was under the impression we want to store all binaries outside of repo. Will fix that.

Comment (jakub-gonet) [reply .gitattributes:7]: https://github.com/expo/expo/pull/13416#discussion_r660148526

**Comment by GollyJer** [comment]
Should we expect this to be working in expo v43 EAS builds?

**Pull request merged**